config/environments/production.rb

```diff
@@ -1,3 +1,4 @@
+require Rails.root.join("config", "mailer")
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Rails/FilePath: Please use Rails.root.join('path/to') instead.